apps/billing/package.json

```diff
@@ -5,8 +5,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "dev": "pnpm dlx trigger.dev@latest dev",
-    "trigger:deploy": "pnpm dlx trigger.dev@latest deploy",
+    "dev": "pnpm dlx trigger.dev@latest dev --skip-update-check",
+    "trigger:deploy": "pnpm dlx trigger.dev@latest deploy --skip-update-check",
```

Comment:
There were some version conflicts when I updated to the latest trigger version, so I skipped their update check